web/models/stable_diffusion/main.py

```diff
@@ -352,6 +366,9 @@ def stable_diff_inf(
     pil_images = [Image.fromarray(image) for image in images]
     out_img = pil_images[0]
 
+    if (args.enable_rtc_profiling):
```

Comment:
Did you test this? At least test running this file?

Reply:
The args are present in stable_args which is a different file.

Reply:
stable_args is one of the files I changed with this. I made some updates, would you mind checking them?